docs/block-processing/slog.md

```diff
@@ -0,0 +1,68 @@
+Slog is a part of block processing that does things which don't fit in any other part (other parts being `Toil` in `txp`, `Poll` in `update`, `Toss` in `ssc`, and `lrc`).
+
+## What it does
+
+Slog is responsible for the following (see explanations below):
+
+* Pure verification
+* Ensuring that leaders from genesis blocks match leaders from LRC
+* State maintenance:
+    * Current tip
+    * `lastBlkSlots`, a list of slots of the last `blkSecurityParam` blocks
+    * Max seen difficulty
+    * Forward links
+    * `inMainChain` flags
+* Calling callbacks (`BListener`)
+
+Pure verification is *TODO DESCRIBE*
+
+Genesis blocks do not contain any information that cannot be derived from elsewhere, but we still store them as physical entities for convenience. Here we want to ensure that the list of leaders that is stored within genesis blocks is the same as the one computed by LRC based on the previous epoch.
```

Comment:
I know markdown doesn't care, but why don't you? I talk about line wrapping.

Reply:
>do not contain anything that cannot be derived from elsewhere

"can be derived from the chain state"

Reply:
Ok. I'll be using `pandoc` for autoformatting, specifically

```
pandoc --from=markdown --to=markdown_github+yaml_metadata_block-hard_line_breaks \
            --columns=80 --atx-headers --normalize \
            slog.md -o slog.md
```

(Just a note for myself, mostly.)

Reply:
> "can be derived from the chain state"
I believe "cannot" is the correct word here, since genesis blocks contain *only* information that *can* be derived from the chain state — thus, they contain *no* information that *cannot* be derived from the chain state.